docs/pages/custom-builds/schema.mdx

```diff
@@ -768,7 +768,7 @@ Behind the scenes, it uses:
 - Series of `run` to execute `maestro test` for each of the provided flows
 - [`eas/upload_artifact`](#easupload_artifact) to upload Maestro test artifacts as build artifact
 
-> **Warning** We have observed that Maestro tests often time out if run on images with Xcode 15.0 or 15.2. Xcode 15.1 and Xcode 14.3.1 seem to be unaffected.
+> **Warning** We have observed that Maestro tests often time out if run on images with Xcode 15.0 or 15.2. Please use the `latest` image to avoid any issues.
```

Comment:
```suggestion
> **Warning** We have observed that Maestro tests often time out if run on images with Xcode 15.0 or 15.2. Use the `latest` image to avoid any issues.
```